<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameSetupForm</class>
 <widget class="QWidget" name="FrameSetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="dLLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data length</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dLTextbox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dataLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataTextbox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send frame</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
